tools/bridge/bridge/event_fetcher.py

```diff
@@ -56,7 +62,7 @@ def __init__(
         self.max_reorg_depth = max_reorg_depth
 
         # Fetching starts always one block after the last one.
```

Comment:
The way it's implemented, it's one block BEFORE the configured one

Reply:
Just starting at `start_block_number` should be fine, considering that there will never be any transfer events in the genesis block.

Reply:
:heavy_check_mark: